app/application.py

```diff
@@ -52,6 +51,24 @@ def hello_world():
     return render_template('_temp.html', _theme='default')
 
 
+def build_survey_data(statusFilter):
+
+    # TODO - Derive the Party Id
+    party_id = "3b136c4b-7a14-4904-9e01-13364dd7b972"
+
+    # TODO - Add security headers
+    # headers = {'authorization': jwttoken}
+    headers = {}
+
+    # Call the API Gateway Service to get the To Do survey list
```

Comment:
Put a #TODO for the 'todo' in the URL. That has to come out, but we might forget about it. If you put a #TODO it will always be picked up by the lint checker and/or pycharm! :-) ....
Or you could stick this as a variable in the config file, which will then get changed when that piece of work gets complete.... Yesh-of-coursh!